applications/GeoMechanicsApplication/custom_conditions/README.md

```diff
@@ -0,0 +1,141 @@
+# Microclimate Boundary Condition
+A microclimate boundary condition refers to the specific set of environmental factors and conditions that influence the climate within a small, localized area, known as a microclimate. Unlike the broader regional or global climate, microclimates are unique climatic conditions that can vary significantly within a relatively small space.
+
+Microclimate boundary conditions are determined by factors such as:
+
+- Air temperature
+- Solar radiation
+- Humidity
+- Precipitation
+- Wind speed
+
+## Radiation
+The net radiation $R_n$ $\mathrm{[W/m^2]}$ is calculated by:
+
+$$ R_n = R^s + R^{l,ab} - R^{l,em} $$
```

Comment:
If I understand it correctly: the short wave radiation is the incoming solar radiation and the long wave radiation is the re-mission from the atmosphere. While indeed the atmospheric radiation is longer wave-length, I think it is confusing to have wave-length as the distinctive property (e.g. the solar radiation also has quite some long wave-lengths following Plancks distribution).

Unless this is a really standard way of writing it in the climate field, I would propose to rename to 
- $R_{s}$ - absorbed solar radiation
- $R_{atm}$ - absorbed atmospheric radiation
- $R_{em}$ - emitted black/grey body radiation from the soil

I'm curious what you think about this!